pkg/sql/show_tenant.go

```diff
@@ -171,7 +172,12 @@ func (n *showTenantNode) startExec(params runParams) error {
 				// replication did not complete the initial scan yet.
 				n.tenantStatus = initReplication
 			} else {
-				n.tenantStatus = replicating
+				progress := n.replicationInfo.IngestionProgress
+				if progress != nil && !progress.CutoverTime.IsEmpty() {
```

Comment:
If this cutover time is far in the future, our behaviour is going to be to allow the replication to reach that timestamp before actually cutting over. For all that time we will be in state `REPLICATION CUTTING OVER`, does this seem strange to others? Should we instead only show this status when we're in `maybeRevertToCutoverTimestamp`?

Reply:
I don't think we should be in `REPLICATING` when a cutover timestamp is set, because this means that a user runs `ALTER TENANT .. COMPLETE ..` and the status of the tenant doesn't change.
if we want we can have another status such as `CUTOVER PENDING` but not sure it's really important now.

Reply:
I like `REPLICATION PENDING CUTOVER` or something to that effect, I think it may cause alarm to the user if they see `REPLICATION CUTTING OVER` when they explicitly set a time an hour in the future from the replicated time.